modules/docs/arrow-docs/docs/docs/arrow-incubator/typeclasses/monadwriter/README.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: docs-incubator
 title: MonadWriter
-permalink: /docs/arrow/mtl/typeclasses/monadwriter/
+permalink: /arrow/mtl/typeclasses/monadwriter/
 redirect_from:
-  - /docs/typeclasses/monadwriter/
```

Comment:
I don't think these ones should be rewritten, as they are the URLs **from** where the site will redirect an user to this page.

Reply:
IMO we can go two routes on this depending on the context:

* Keep the redirect from `docs` preceded URL and the new one, using an array of paths, as `jekyll-redirect-from` support them.
* If the URLs is gonna be lost either way (maybe if this change was made in a previous version), the `redirect_from` might not be needed anymore.

Reply:
Ok @calvellido , I think the best option would be removing the `redirect_from`.